### PR TITLE
1719 add blazer to find publish service for support 2

### PR DESCRIPTION
**Pull request by avinhurry**
### Checklist

- [ ] Make sure all information from the Trello card is in here
- [ ] Attach to Trello card
- [ ] Rebased main
- [ ] Cleaned commit history
- [ ] Tested by running locally
- [ ] Inform data insights team due to database changes

### CODE REVIEW

File: app/lib/blazer_admin_constraint.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+    admin_user = User.admins.kept.find_by(email: signin_user.email)
+    admin_user.present? && authorised_for_blazer?(signin_user)
+  end
+
+  private
+
+  def authorised_for_blazer?(signin_user)
+    User.with_blazer_access.find_by(email: signin_user.email).present?
+  end
```
Comment (elceebee) [inline app/lib/blazer_admin_constraint.rb:16]:
```suggestion
    blazer_user = User.with_blazer_access.kept.find_by(email: signin_user.email)
    blazer_user.present?
  end
```

Comment (avinhurry) [reply app/lib/blazer_admin_constraint.rb:16]: Thanks! This is a lot neater, i've updated in this commit: https://github.com/DFE-Digital/publish-teacher-training/pull/4277/commits/c123a3516080f02deb075b3f3fbe690761662a60

Had to open a new PR.

**Comment by avinhurry** [comment]
Superseded by: https://github.com/DFE-Digital/publish-teacher-training/pull/4277

**Comment by avinhurry** [comment]
Superseded by: https://github.com/DFE-Digital/publish-teacher-training/pull/4277

**Comment by avinhurry** [comment]
[Determine the security requirements and implications of using Blazer in Apply Support Console](https://trello.com/c/sqtnQIBI/103-determine-the-security-requirements-and-implications-of-using-blazer-in-apply-support-console)

**Pull request closed without merging**